src/main/msp/msp.c

```diff
@@ -1354,6 +1354,7 @@ case MSP_NAME:
         sbufWriteU8(dst, 0);   // was currentControlRateProfile->tpa_rate
         sbufWriteU8(dst, currentControlRateProfile->thrMid8);
         sbufWriteU8(dst, currentControlRateProfile->thrExpo8);
+        sbufWriteU8(dst, currentControlRateProfile->thrHover8);
```

Comment:
must be last

Reply:
Yes MSP always LILO (or whatever), thank you for reminding.

Reply:
- someone marked resolved -- it was not.  this should be moved under rates_type with accompanying comment.
- the sbufRead must also be moved.
- the controlRateConfig_t and controlRateConfig_s must also be changed.